NIPTool/API/external/api/deps.py

```diff
@@ -67,20 +62,18 @@ def get_password_hash(password):
     return pwd_context.hash(password)
```

Comment:
Specify type hint for argument `password` and return type to str

(Could not do it on GitHub)